cypress/support/cypress-support/src/utils.js

```diff
@@ -736,6 +737,8 @@ class FireLog {
   assert(expression, message) {
     assert(expression, message);
   }
+
+  info(message) {}
```

Comment:
Can add a comment on top of this function on why it is written like this, otherwise a dev might be confused on seeing an empty function definition.